solidity/dashboard/src/connectors/explorer-mode-connector.js

```diff
@@ -0,0 +1,57 @@
+import { AbstractConnector } from "./abstract-connector"
+import { WALLETS } from "../constants/constants"
+import { getWsUrl } from "./utils"
+import Web3ProviderEngine from "web3-provider-engine"
+import WebsocketSubprovider from "web3-provider-engine/subproviders/websocket"
+import ExplorerModeSubprovider from "./explorerModeSubprovider"
+import { EventEmitter } from "events"
+import { Subprovider } from "@0x/subproviders"
+
+export class ExplorerModeConnector extends AbstractConnector {
+  /** @type {string} To store selected account by user. */
+  selectedAccount = ""
+
+  constructor() {
+    super(WALLETS.EXPLORER_MODE.name)
+    this.explorerModeSubprovider = Object.assign(
+      ExplorerModeSubprovider.prototype,
+      Subprovider.prototype,
+      EventEmitter.prototype
+    )
```

Comment:
Hmmm I think it would be better to handle it in `./explorerModeSubprovider`.
```
// explorerModeSubprovider.js
import { EventEmitter } from "events"
import { Subprovider } from "@0x/subproviders"

class ExplorerModeSubprovider extends Subproivder {...}

Object.assign(ExplorerModeSubprovider.prototype, EventEmitter.prototype)

export default ExplorerModeSubprovider
```